pkg/system/phase2_creating.go

```diff
@@ -519,6 +546,44 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &bucketLogVolumeMounts)
 			}
+
+			if r.NooBaa.Spec.BucketNotifications.Enabled {
+				notificationVolumeMounts := []corev1.VolumeMount{{
+					Name:      "notif-vol",
```

Comment:
take out to a constant to guarantee the same name here and in the volume. also, notification-volume sounds better.